frontend/src/components/predictiontools/LineChart.jsx

```diff
@@ -17,13 +18,40 @@ ChartJS.register(
   LineElement,
   Title,
   Tooltip,
-  Legend
+  Legend,
+  annotationPlugin
 );
 // above imports all using Chart-js library!
 
 // dummay data, mostly just to understand how to format my TC #2 data!
 
-const LineChart = ({ portfolioData, predictionData }) => {
+const LineChart = ({ portfolioData, predictionData, predictedShifts }) => {
+  const buildShifts = (dates) => {
+    const annotations = {};
+    dates.forEach((val, i) => {
+      annotations[`line${i}`] = {
+        type: "line",
+        scaleID: "x",
+        value: val.date,
+        borderColor: "red",
+        borderWidth: 2,
+        label: {
+          display: true,
+          enabled: true,
+          content: `${val.name}'s ${val.description}`,
```

Comment:
nit: why make this a property of the shift object instead of hard-coding it here? description will always be "earnings release", right?